packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1064,9 +1065,15 @@ export default class Meetings extends WebexPlugin {
         .fetchInfoOptions(destination, type)
         // Catch a failure to fetch info options.
         .catch((error) => {
-          LoggerProxy.logger.info(
-            `Meetings:index#create --> INFO, unable to determine info options: ${error.message}`
+          LoggerProxy.logger.error(
+            `Meetings:index#create --> ERROR, unable to determine info options: ${error.message}`
           );
+          if (error.error === '101') {
```

Comment:
Can we create a new file for error enums so we can refer to them in various places? We can use them in existing code as well for various errors and this will help us in documenting and enhacing the SDK

Reply:
Sure @sreenara, let me have a look at it.

Reply:
@sreenara, Addressed,